src/prompts/getLang.ts

```diff
@@ -0,0 +1,25 @@
+import chalk from 'chalk';
+import prompts from 'prompts';
+import { LANGS } from '../constants';
+import { onCancel } from '../helpers';
+import { PromptFunction } from '../types';
+
+export const getLang: PromptFunction = async function (config) {
+  const { lang } = await prompts(
+    {
+      type: config.lang ? null : 'select',
+      name: 'lang',
+      message: chalk.bold.white('Select a lang:'),
+      choices: LANGS.map((lang) => {
+        const langColor = lang.color;
+        return {
+          title: langColor(lang.name),
```

Comment:
```suggestion
        return {
          title: lang.color(lang.name),
```

есть причина создавать отдельную переменную здесь?